provisioning/standalone/jupyterlab/README.md

```diff
@@ -0,0 +1,11 @@
+# Jupyterlab
+
+This is a **jupyterlab** container, supporting **CrownLabs standalone applications**.
```

Comment:
Should add one/two sentences about the most relevant elements (Is that a wrapper around an official image? Is rewrite required? Are there any important aspects to pay attention to?). If there is nothing relevant to report, just say so (e.g., it works out of the box and no particular configuration is possible/necessary).

Reply:
![image](https://user-images.githubusercontent.com/33266330/168756147-e746afbc-fd0d-4fa7-82bf-02afd786c7e2.png)
I added those lines

Reply:
Looks great. Would only add that it does not require rewrite to work.